### DISCUSSION TITLE
Add support for link remote emulation to play multiplayer on games like pokemon on gb,gbc,gba etc. over wifi.

### DISCUSSION THREAD

**Discussion by Flamehawk7** [OnionUI/Onion#990] [Ideas]
Is there any person here with some expertise on the onion os to tell us if the possibility to add this feature exists? Cause i am new here and i have no idea.

Respect to the devs.

**Comment by XK9274** [comment]
This was finalised in https://github.com/OnionUI/Onion/pull/1026 for Hotspot and is live now pending changes for the next rls/RC and will potentially be expanded to solo trading for people with 1 MM/MM+ and potentially global if we can figure out the logistics of it.
